recipes/qpdf/all/conandata.yml

```
@@ -0,0 +1,11 @@
+sources:
+  "11.1.1":
+    url: [
+      "https://github.com/qpdf/qpdf/archive/refs/tags/v11.1.1.tar.gz",
+    ]
+    sha256: "785edab622a1bc7e25e1537ad2c325005d48c5c7957f7abedff405deb80fa59a"
```

Let's skip on the mirror syntax

```suggestion
    url: "https://github.com/qpdf/qpdf/archive/refs/tags/v11.1.1.tar.gz"
    sha256: "785edab622a1bc7e25e1537ad2c325005d48c5c7957f7abedff405deb80fa59a"
```